src/etl/do_etl.py

```diff
@@ -37,141 +43,159 @@ class DOETL(ETL):
                         if "DOID:" in alt_id:
-                            secondaryId = {
-                                "primary_id": k,
+                            secondary_id = {
+                                "primary_id": key,
                                 "secondary_id": alt_id
                             }
-                            do_altids_list.append(secondaryId)
+                            do_alt_ids_list.append(secondary_id)
 
                 if "xrefs" in node["meta"]:
-
                     o_xrefs = node["meta"].get('xrefs')
                     if o_xrefs is not None:
-                        for xrefIdDict in o_xrefs:
-                            xrefId = xrefIdDict["val"]
-                            if ":" in xrefId:
-                                local_id = xrefId.split(":")[1].strip()
-                                prefix = xrefId.split(":")[0].strip()
-                                complete_url = ETLHelper.get_complete_url_ont(local_id, xrefId)
-                                generated_xref = ETLHelper.get_xref_dict(local_id, prefix, "ontology_provided_cross_reference", "ontology_provided_cross_reference", xrefId, complete_url, xrefId + "ontology_provided_cross_reference")
+                        for xref_id_dict in o_xrefs:
+                            xref_id = xref_id_dict["val"]
+                            if ":" in xref_id:
+                                local_id = xref_id.split(":")[1].strip()
+                                prefix = xref_id.split(":")[0].strip()
+                                complete_url = ETLHelper.get_complete_url_ont(local_id, xref_id)
+                                generated_xref = ETLHelper.get_xref_dict(\
+                                        local_id,
+                                        prefix,
+                                        "ontology_provided_cross_reference",
+                                        "ontology_provided_cross_reference",
+                                        xref_id,
+                                        complete_url,
+                                        xref_id + "ontology_provided_cross_reference")
                                 generated_xref["oid"] = ident
                                 xrefs.append(generated_xref)
-                        else:
+                        else: #TODO Need to make sure this else is correct
                             if ":" in o_xrefs:
                                 local_id = o_xrefs.split(":")[1].strip()
                                 prefix = o_xrefs.split(":")[0].strip()
                                 complete_url = ETLHelper.get_complete_url_ont(local_id, o_xrefs)
-                                generated_xref = ETLHelper.get_xref_dict(local_id, prefix, "ontology_provided_cross_reference", "ontology_provided_cross_reference", o_xrefs, complete_url, o_xrefs)
+                                generated_xref = ETLHelper.get_xref_dict(\
```

Comment:
can the first item in the list be in line with the function call, I find this more readable than the \